integration/angular_cli/e2e/src/app.e2e-spec.ts

```diff
@@ -8,9 +8,9 @@ describe('workspace-project App', () => {
     page = new AppPage();
   });
 
-  it('should display welcome message', () => {
-    page.navigateTo();
-    expect(page.getTitleText()).toEqual('integration-project app is running!');
+  it('should display welcome message', async () => {
+    await page.navigateTo();
+    await expect(await page.getTitleText()).toEqual('integration-project app is running!');
```

Comment:
The outer await should not be needed here and in the other tests.

Reply:
The `tsconfig.json.template` file in `packages/schematics/angular/e2e/files` specifies `jasminewd2` in types. The method signatures in these type definitions return Promises. So I believe this change is needed.

Reply:
It returns a promise because it's a generic, therefore if you `await` the return type is no longer a promise.

```ts
declare function expect<T>(actual: ArrayLike<T>): jasmine.ArrayLikeMatchers<T>;
```